modules/ocf_decal/manifests/website.pp

```diff
@@ -0,0 +1,43 @@
+class ocf_decal::website {
+
+  file {
+    ['/srv/www', '/srv/www/decal']:
+      ensure => directory,
+      owner  => ocfdecal,
+      group  => www-data,
+      mode   => '0755',
```

Comment:
Why do you need to specify the mode here? Isn't this the default for directories?

Reply:
Yeah it is, I'll remove this